waterlog/src/reducers/SegmentEventsReducer.js

```diff
@@ -0,0 +1,38 @@
+import {
+    FETCH_SEGMENT_EVENTS_BEGIN,
+    FETCH_SEGMENT_EVENTS_SUCCESS,
+    FETCH_SEGMENT_EVENTS_FAILURE
+} from '../actions/Types';
+
+const initialState = {
+    items: [],
+    loading: true,
+    error: null
+}
+
+export default function (state = initialState, action) {
+    switch (action.type) {
+        case FETCH_SEGMENT_EVENTS_BEGIN:
+            return {
+                ...state,
+                loading: true,
+                error: null
+            };
+        case FETCH_SEGMENT_EVENTS_SUCCESS:
+            return {
+                ...state,
+                loading: false,
+                items: action.payload.events,
+                error: null
+            }
+
+        case FETCH_SEGMENT_EVENTS_FAILURE:
+            return {
+                ...state,
+                loading: false,
+                error: action.payload.error
+            }
+        default:
+            return state;
+    }
+} 
```

Comment:
@LisaQ04 this is nothing new, @RashadAkoodieRabbit already commented on this, `add a new line at end of file` 
which such comments check for similar mistakes in other files too. I see you fixed where he mentioned it but you don't check for the same mistake in other files too

Reply:
sure I fixed it.

Reply:
but this file still doesn't have a new line?
![image](https://user-images.githubusercontent.com/35101587/52703181-6cdc2280-2f86-11e9-81e4-f4bcba1766b5.png)